### ISSUE TITLE
Fail to build with zig master 

### ISSUE THREAD

**Issue by Avokadoen** [Snektron/vulkan-zig#44]
Seems like align is a illegal operation on packed structs now

Complier error example:

```txt
error: unable to override alignment of packed struct fields
    externally_synchronized_bit_ext: bool align(@alignOf(Flags)) = false,
```

Temporary monkey patch of removing the following lines works for this:
Line 869-871 in ``generator/vulkan/render.zig``:
```zig
if (bitpos == 0) { // Force alignment to integer boundaries
   try self.writer.print("align(@alignOf({s})) ", .{flags_type});
}
```

So it seems like the default behavior of packed structs in zig is to align to smallest int that fits the given struct, but I have yet to find the commit and/or issue discussing this change so this might introduce subtle bugs.

Also ...

GetImageViewAddressNVXError in vk.zig gets two identical Unknown members in its enum. 

The following fixes it, but comes at the cost of having to check each error write ...

Line 1341-.. in render.zig
 ```zig
        fn renderErrorSet(self: *Self, errors: []const []const u8) !void {
            try self.writer.writeAll("error{");
            for (errors) |name| {
                if (std.mem.endsWith(u8, name, "UNKNOWN")) continue;
```

**Comment by Snektron** [comment]
This problem should already be resolved since 470615632d4938ba6976ab4f56b8bf991a383360

**Comment by Avokadoen** [comment]
Hi, did you also fix the duplicate unknown. Just added it to the description 😄

**Comment by Snektron** [comment]
Yes, that should be fixed since aac20d107a9c04a09dd756d7b5e1edacc0cf23a0

**Comment by Avokadoen** [comment]
It is indeed fixed! Sorry for the trouble. Time to update my dependencies ☠️
